README.md

```diff
@@ -11,6 +11,9 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 Kyma Istio Operator is a component of the Kyma runtime that handles the management and configuration of the Istio service mesh. Within Kyma Istio Operator, [Istio Controller](/docs/user/00-10-overview-istio-controller.md) is responsible for installing, uninstalling, and upgrading Istio.
 
+**Istio version:** 1.21.2  
```

Comment:
```suggestion
The latest release includes the following versions of Istio and Envoy:
**Istio version:** 1.21.2  
```

Reply:
I'd also add this information to the readme in the /user folder
https://github.com/kyma-project/istio/blob/main/docs/user/README.md

Reply:
the one in the /user folder is displayed on the website and the other one is the main repo's readme, so I guess it's better to have duplicated. I totally get your point, though :)